<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
 
	 
	background-color: rgb(85, 170, 255);
}  


/*Estilos para el botón*/
	QPushButton{
		background-color: #ff5722;
		border-radius: 4px;
		color: #fff;
		font-family: 'Roboto';
		font-size: 12px;
	}


estadoled{
	 
 
	background-color: rgb(255, 255, 255);

}

#btn_Avanzar{
		 
	background-color: rgb(85, 0, 255);
		border-radius:10px;
		color: #fff;
		font-family: 'Roboto';
		font-size: 12px;
}

#btn_Retroceder{
		 
	background-color: rgb(85, 0, 255);
		border-radius:10px;
		color: #fff;
		font-family: 'Roboto';
		font-size: 12px;
}

#btn_Derecha{
		 
	background-color: rgb(85, 0, 255);
		border-radius:10px;
		color: #fff;
		font-family: 'Roboto';
		font-size: 12px;
}

#btn_Izquierda{
		 
	background-color: rgb(85, 0, 255);
		border-radius:10px;
		color: #fff;
		font-family: 'Roboto';
		font-size: 12px;
}

#btn_Parar{
		 
	background-color: rgb(85, 0, 255);
		border-radius:10px;
		color: #fff;
		font-family: 'Roboto';
		font-size: 12px;
}</string>
  </property>
  <widget class="QLabel" name="Titulo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sistema de Control led para Arduino</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>106</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado del Led 13</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>481</width>
     <height>34</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="btn_onLed">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>ON Led  13</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_offLed">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>OFF Led  13</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="estadoLed">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>NEUTRO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>227</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Arduino está:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cArduino">
    <property name="text">
     <string>¡En espera!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocidad del Motor</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Avanzar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Avanzar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Retroceder">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Retroceder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Derecha">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Derecha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Izquierda">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Izquierda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Parar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Parar</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <zorder>splitter</zorder>
  <zorder>splitter_2</zorder>
  <zorder>Titulo</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>verticalSlider</zorder>
  <zorder>btn_Avanzar</zorder>
  <zorder>btn_Retroceder</zorder>
  <zorder>btn_Derecha</zorder>
  <zorder>btn_Izquierda</zorder>
  <zorder>btn_Parar</zorder>
  <zorder>progressBar</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
